app/javascript/react/screens/App/Plan/index.js

```diff
@@ -1,17 +1,23 @@
 import { connect } from 'react-redux';
 import Plan from './Plan';
 import * as PlanActions from './PlanActions';
+import * as NotificationActions from '../common/NotificationList/NotificationListActions';
 
 import reducer from './PlanReducer';
 
 export const reducers = { plan: reducer, form: {} };
 
-const mapStateToProps = ({ plan }, ownProps) => ({
+const mapStateToProps = ({ plan }, ownProps, notifications) => ({
   ...plan,
-  ...ownProps.data
+  ...ownProps.data,
+  ...notifications
 });
 
 const mergeProps = (stateProps, dispatchProps, ownProps) =>
   Object.assign(stateProps, ownProps.data, dispatchProps);
 
-export default connect(mapStateToProps, PlanActions, mergeProps)(Plan);
+export default connect(
+  mapStateToProps,
+  Object.assign(PlanActions, NotificationActions),
+  mergeProps
+)(Plan);
```

Comment:
In the same vein of thought in my last comment, we don't actually need to inject the `notifications` state nor actions into `Plan`.

We provide `Overview` with the NotificationActions b/c we actually need to call `addNotificationAction()` from within `Overview.js` (so with this in mind, we technically don't need to import all the actions, just the one we need, but a minor detail imo)

But Plan, nor any of its nested components, need access to any of the Notification stuff, yet

Reply:
*yet* was my thought as well...